release.sh

```diff
@@ -1,16 +1,21 @@
 #!/bin/bash
-rm -rf ~/Downloads/pk-release/
-mkdir ~/Downloads/pk-release/
+
+version=2.3.2
```

Comment:
Maybe this should be set by using arguments?
```
./release.sh 2.3.2
```
And add set the `nounset` flag to check for unset variables.

Reply:
This is only a temporary solution until https://github.com/exasol/release-droid/issues/228 is fixed. I don't want to invest too much time into this workaround.